apps/elasticsearch/templates/volumes/bootstrap.yml.j2

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
This volume should be created only if `elasticsearch_security_enabled` is set to true, don't you think ?

Reply:
Yes. I wanted it to pre-exist in case the user wants to switch from a non-secure to secure mode (as volumes are created while initializing the project).

Reply:
This should maybe added to the documentation ?